Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -355,3 +355,21 @@ private extension SPAppDelegate {
         verificationViewController = nil
     }
 }
+
+
+// MARK: - Scroll position cache
+//
+extension SPAppDelegate {
+    @objc
+    func cleanupScrollPositionCache() {
+        let allNotes = SPObjectManager.shared().notes()
+        let allIdentifiers: [String] = allNotes.compactMap {
+            guard let note = $0 as? Note, !note.deleted else {
+                return nil
+            }
+
+            return note.simperiumKey
+        }
```

Comment:
Suggestion!!

1. Edit `SPObjectManager.h` and set the collection types:
```
- (NSArray<Note *> *)notes;
- (NSArray<Tag *> *)tags;
```

2. Then you can do something like this!
```
let allIdentifiers: [String] = allNotes.compactMap { note in
   note.deleted ? nil : note.simperiumKey
}
```

Reply:
Great suggestion. Love it.